duckduckgo/content-scope-scripts
# ntp: customizer polish 💅

**Asana Task/Github Issue:** https://app.asana.com/0/1201141132935289/1208996908465307/f

## Description

- [x] [[NTP 💅] Background should not animate as window expands ](https://app.asana.com/0/1201141132935289/1208996908465307/f)
- [x] [[NTP 💅] Gradient tiles on dark background](https://app.asana.com/0/1201141132935289/1208996908465317/f)
- [x] [[NTP 💅] sub-nav in the drawer should animate between sections](https://app.asana.com/0/0/1208996912816680/f) 
- [x] [[NTP 💅] user images screen missing placeholders](https://app.asana.com/0/0/1209005646748698/f)
- [x] [Support auto open](https://app.asana.com/0/0/1208996912816686/f) 
- [x] [[NTP 💅] Customizer drawer elements to match designs](https://app.asana.com/0/0/1208996908465313/f)

## Testing Steps

- Use the customizer drawer here https://deploy-preview-1336--content-scope-scripts.netlify.app/build/pages/new-tab/?customizerDrawer=enabled&userImages
- Verify the page continues to work as expected when the drawer is DISABLED https://deploy-preview-1336--content-scope-scripts.netlify.app/build/pages/new-tab/?customizerDrawer=disabled

> [!NOTE]
> When viewing as a URL you'll get some some additionally latency on images, that won't be present in the browser.

## Checklist


*Please tick all that apply:*
